truss-chains/truss_chains/deployment/code_gen.py

```diff
@@ -458,6 +458,19 @@ def _gen_load_src(chainlet_descriptor: definitions.ChainletAPIDescriptor) -> _So
     return _Source(src=src, imports=imports)
 
 
+def _gen_health_check_src(
+    health_check: definitions.HealthCheckAPIDescriptor,
+) -> _Source:
```

Comment:
nit: wondering if it could be worth it to show in a docstring what the expected output is ?

Reply:
I added the expected output to the comment in `truss-chains/truss_chains/remote_chainlet/model_skeleton.py`